src/main/java/seedu/address/model/tag/TagSet.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.model.tag;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TagSet {
```

Comment:
Does this class actually represent a set of tags? Or is it more of a utility class?